ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -375,14 +376,14 @@ class ( ShowLedgerState (LedgerTables l)
 
   pureLedgerTables ::
        (forall k v.
-            Ord k
+            (Ord k, Eq v)
```

Comment:
I'm unsure why are these Eq constraints propagated if we are not comparing the values in the end?